Formula/opencl-clhpp.rb

```diff
@@ -0,0 +1,36 @@
+class OpenclClhpp < Formula
+  desc "OpenCL API C++ bindings"
+  homepage "https://github.com/KhronosGroup/OpenCL-CLHPP"
+  url "https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2022.09.30.tar.gz"
+  sha256 "999dec3ebf451f0f1087e5e1b9a5af91434b4d0c496d47e912863ac85ad1e6b2"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/OpenCL-CLHPP.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "opencl-headers"
+  depends_on "opencl-icd-loader"
```

Comment:
it should not depend on ICD loader

Reply:
How so? It says this in the README

Reply:
I am sorry, I have just now realized one needs them for building the examples and for the project itself